development/sass/top-nav.scss

```diff
@@ -0,0 +1,72 @@
+.navbar {
```

Comment:
actually it looks like you forgot to place this code in the other file. This looks like the accurate code.